src/lib/metricsMgr/MetricsManager.cpp

```diff
@@ -130,16 +133,22 @@ void MetricsManager::add(const std::string& srv, const std::string& subServ, con
 
   //
   // Exclude the first '/' from the Sub Service
+  // But, only if it starts with a '/'
   //
-  const char* subService = &subServ.c_str()[1];
+  const char* subService = subServ.c_str();
+
+  if (subService[0] == '/')
+  {
+    subService = &subService[1];
+  }
```

Comment:
Service path not starting with '/' is erroneous, and the request is saved in the metrics map with the erroneous service path. If we strip the first letter for this kind of requests it gets weird ...

NTC